eternalcore-api/src/main/java/com/eternalcode/core/feature/jail/event/JailReleaseEvent.java

```diff
@@ -0,0 +1,42 @@
+package com.eternalcode.core.feature.jail.event;
+
+import org.bukkit.event.Cancellable;
+import org.bukkit.event.Event;
+import org.bukkit.event.HandlerList;
+
+import java.util.UUID;
+
+public class JailReleaseEvent extends Event implements Cancellable {
+
+    private static final HandlerList HANDLER_LIST = new HandlerList();
+
+    private final UUID uuid;
+    private boolean cancelled = false;
```

Comment:
The primitive logical value has a default value of false

```suggestion
    private boolean cancelled;
```